bin/build-plugin-zip.sh

```diff
@@ -25,34 +35,50 @@ elif ! git diff --cached --exit-code > /dev/null; then
 fi
 if [ ! -z "$changed" ]; then
 	git status
-	echo "ERROR: Cannot build plugin zip with dirty working tree."
-	echo "       Commit your changes and try again."
-	exit 1
+	error "ERROR: Cannot build plugin zip with dirty working tree. ☝️
+       Commit your changes and try again."
+	exit
```

Comment:
I'm not saying we shouldn't make this change, but it's yet another thing that makes me dislike `npm`, that we have to make concessions and break from the Unix pattern.

(Perhaps a more correct fix would be to keep `exit 1` and use some process as membrane between this script and `npm run`, e.g. `"package-plugin": "./bin/build-plugin-zip.sh | silent"` or `"package-plugin": "silence-errors ./bin/build-plugin-zip.sh"`. Anyway, just a digression, ignore this old soul.)

Reply:
I actually totally agree… I mean, in reality, it's a bash script and shouldn't be run from npm anyway. I should keep the exit codes and just change the docs to say `run ./bin/build-plugin-zip.sh`.

Would you be okay with that change? We can leave the node script in for people, but encourage straight bash usage 🤷‍♂️